vllm/logger.py

```diff
@@ -26,15 +36,23 @@ def format(self, record):
 _default_handler = None
 
 
+def _get_default_logging_level():
+    verbosity = os.getenv("VLLM_VERBOSITY", "info").lower()
+    
+    try:
+        return log_levels[verbosity]
+    except KeyError:
+        raise ValueError(f"Unknown log level: {verbosity}")
+
 def _setup_logger():
-    _root_logger.setLevel(logging.DEBUG)
+    _root_logger.setLevel(_get_default_logging_level())
     global _default_handler
     if _default_handler is None:
         _default_handler = logging.StreamHandler(sys.stdout)
         _default_handler.flush = sys.stdout.flush  # type: ignore
-        _default_handler.setLevel(logging.INFO)
+        _default_handler.setLevel(logging.DEBUG)
```

Comment:
The default handler to down to `debug`. But the root is defining the effective level.